### DISCUSSION TITLE
Optimize prompt for Claude3.5 sonnet to save tokens.

### DISCUSSION THREAD

**Discussion by alextyhwang** [cline/cline#502] [Feature Requests]
When the action finishes, it regularly spits out a summary of its actions. I find it long and unnecessary.

Example:
 
<img width="408" alt="Screenshot 2024-10-11 at 10 17 09 AM" src="https://github.com/user-attachments/assets/493e6edd-95f0-4368-9495-a17913690ef6">

**Comment by ctwhome** [comment]
In my case, this is fundamental to wrapping up the entire changes and share it on an issue, for example. It is s fundamental part for me.
